python/lsst/pipe/tasks/calibrateImage.py

```diff
@@ -670,8 +711,15 @@ def _measure_aperture_correction(self, exposure, bright_sources):
             necessary for point source determination for the aperture correction
             calculation.
         """
-        result = self.measure_aperture_correction.run(exposure, bright_sources)
-        exposure.setApCorrMap(result.apCorrMap)
+        norm_ap_corr_map = exposure.info.getApCorrMap()
```

Comment:
This is directly available on the exposure as a property:
```suggestion
        norm_ap_corr_map = exposure.apCorrMap
```